README.md

```
@@ -168,6 +177,325 @@ tests/
+```
+
+프로덕션 배포를 위한 최적화된 빌드:
+
+```bash
+# 프로덕션 빌드 (minified)
+npm run build
+```
+
+### 파일 구조
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)